File: openapi-generator/java-play-framework/app/apimodels/InlineObject24.java
package apimodels;

import java.util.ArrayList;
import java.util.List;
import com.fasterxml.jackson.annotation.*;
import java.util.Set;
import javax.validation.*;
import java.util.Objects;
import javax.validation.constraints.*;
/**
 * InlineObject24
 */
@javax.annotation.Generated(value = "org.openapitools.codegen.languages.JavaPlayFrameworkCodegen", date = "2019-03-01T05:34:42.270Z[GMT]")

@SuppressWarnings({"UnusedReturnValue", "WeakerAccess"})
public class InlineObject24   {
  @JsonProperty("groupId")
  private Long groupId;

  @JsonProperty("sensors")
  private List<Long> sensors = new ArrayList<>();

  public InlineObject24 groupId(Long groupId) {
    this.groupId = groupId;
    return this;
  }

   /**
   * Group ID to query.
   * @return groupId
  **/
  @NotNull
  public Long getGroupId() {
    return groupId;
  }

  public void setGroupId(Long groupId) {
    this.groupId = groupId;
  }

  public InlineObject24 sensors(List<Long> sensors) {
    this.sensors = sensors;
    return this;
  }

  public InlineObject24 addSensorsItem(Long sensorsItem) {
    sensors.add(sensorsItem);
    return this;
  }

   /**
   * List of sensor IDs to query.
   * @return sensors
  **/
  @NotNull
  public List<Long> getSensors() {
    return sensors;
  }

  public void setSensors(List<Long> sensors) {
    this.sensors = sensors;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    InlineObject24 inlineObject24 = (InlineObject24) o;
    return Objects.equals(groupId, inlineObject24.groupId) &&
        Objects.equals(sensors, inlineObject24.sensors);
  }

  @Override
  public int hashCode() {
    return Objects.hash(groupId, sensors);
  }

  @SuppressWarnings("StringBufferReplaceableByString")
  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class InlineObject24 {\n");
    
    sb.append("    groupId: ").append(toIndentedString(groupId)).append("\n");
    sb.append("    sensors: ").append(toIndentedString(sensors)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: openapi-generator/dart/lib/model/inline_object24.dart
part of openapi.api;

class InlineObject24 {
  /* Group ID to query. */
  int groupId = null;
  /* List of sensor IDs to query. */
  List<int> sensors = [];
  InlineObject24();

  @override
  String toString() {
    return 'InlineObject24[groupId=$groupId, sensors=$sensors, ]';
  }

  InlineObject24.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    if (json['groupId'] == null) {
      groupId = null;
    } else {
          groupId = json['groupId'];
    }
    if (json['sensors'] == null) {
      sensors = null;
    } else {
      sensors = (json['sensors'] as List).cast<int>();
    }
  }

  Map<String, dynamic> toJson() {
    return {
      'groupId': groupId,
      'sensors': sensors
    };
  }

  static List<InlineObject24> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineObject24>() : json.map((value) => new InlineObject24.fromJson(value)).toList();
  }

  static Map<String, InlineObject24> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, InlineObject24>();
    if (json != null && json.length > 0) {
      json.forEach((String key, dynamic value) => map[key] = new InlineObject24.fromJson(value));
    }
    return map;
  }
}

